.circleci/config.yml

```diff
@@ -242,8 +242,7 @@ commands: # reusable commands with parameters
       - run:
           name: Run cucumber tests
           command: |
-            TEST=$(circleci tests glob "features/**/*.feature" | circleci tests split --split-by=timings)
-            bundle exec cucumber --profile ci ${TEST}
+            bundle exec cucumber --profile ci features
```

Comment:
Why? Do you want to run all the features (really all of them) in all containers each time?
If the `circleci` program cannot find the tests, it still splits them (I think randomly or any other means) but it splits them

Reply:
It is merely an experiment, I need to find the failing cucumber
<img width="315" alt="Screenshot 2021-03-16 at 15 50 08" src="https://user-images.githubusercontent.com/11672286/111329577-7c4f5d80-866f-11eb-89fe-92ee8566f89a.png">

🚧 in gitmoji means "work in progress" so not final

Reply:
I showed you how to find the failing cucumber
If you want lets talk about that and screenshrare it

Reply:
Removed